R/tm_t_crosstable.R

```diff
@@ -283,7 +283,7 @@ srv_t_crosstable <- function(id, data, reporter, filter_panel_api, label, x, y,
           substitute(
             expr = {
               lyt <- basic_tables %>%
-              split_call %>% # styler: off
+                split_call %>% # styler: off
```

Comment:
summoning @averissimo our lintr guru these days. Is this something that appears only here, or should we revisit the whole package? If yes, can we do it on a separate PR (pointing to this one)?

Reply:
It's odd that it's even here, everything seems to follow the style guide.

Reply:
Spotted this because we touched that line. I think it was the reason why it got not styled. The change was good, the style had been applied and we probably don't need that line.
Haven't thought about the whole package. Checked this quickly and in my local repo (might be not in sync with pre-release branch) I found 7 occurences of `styler: off`. Some of them might be meaningful though.

Reply:
thanks @averissimo . makes sense now!